CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+### 2.1 
+ * **IMPORTANT UPGRADE NOTE FOR CRUNCHY PACKAGE USERS**: In version 2.0, the Crunchy provided extras for node_exporter were split out from the pgmonitor-pg##-extras package. A dependency was kept between these packages to make upgrading easier. For 2.1, the dependency between these packages has been removed. When upgrading from 1.7 or earlier, if you have node_exporter and postgres_exporter running on the same systems, ensure that you install the separate pgmonitor-node_exporters_extras package after the update. See the README for the full package name(s).
+
+ * Minium required versions of software used in pgmonitor have been updated to: Prometheus 2.5.0, Prometheus Alertmanager 0.15.3, postgres_exporter 0.4.7 (enables full PG11 support) & Grafana 5.3.4.
```

Comment:
Typo "Minium" -> "Minimum"

This line required "if you have node_exporter and postgres_exporter running on the same systems," ?
Based on dependency, it will auto pull node_exporter, correct ? If this was the case, it means both were on same serve, agree ?

May be make bullet list for software version for better readability.
* Prometheus 2.5.0
* Prometheus Alertmanager 0.15.3
* postgres_exporter 0.4.7 (enables full PG11 support)
* Grafana 5.3.4.

Reply:
Fixed and will change to bullet list